cypress/components/pager/pager.cy.tsx

```diff
@@ -397,121 +395,100 @@ context("Test for Pager component", () => {
       );
 
       pageSelect().click();
-      selectListWrapper()
-        .find("li")
-        .contains("25")
-        .click()
-        .then(() => {
-          // eslint-disable-next-line no-unused-expressions
-          expect(callback).to.have.been.calledOnce;
-        });
+      selectListWrapper().find("li").contains("25").click();
+
+      cy.get("@onPagination").should("have.been.calledOnce");
     });
 
     it("should call onNext callback when a click event is triggered", () => {
+      const callback: PagerProps["onNext"] = cy.stub().as("onNext");
+
       CypressMountWithProviders(<PagerComponent onNext={callback} />);
 
-      nextArrow()
-        .click()
-        .then(() => {
-          // eslint-disable-next-line no-unused-expressions
-          expect(callback).to.have.been.calledOnce;
-        });
+      nextArrow().click();
+      cy.get("@onNext").should("have.been.calledOnce");
     });
 
-    it.each([["Enter"], ["Space"]])(
+    it.each([keysToTrigger[0], keysToTrigger[1]])(
```

Comment:
suggestion: you could just pass the array here like so, again not a big deal

```suggestion
    it.each(keysToTrigger)(
```